### PR TITLE
Fix possessive adjective in hooks2 "time traveller obsessed with finding his/her descendant"

### LINKED ISSUE #2: Minor inconsistencies with pronouns with sex selection
I've noticed the plot hooks will sometimes begin with the select sex pronounce (male = he/female = she) and then randomly change the opposite, I think it would be easier to keep it neutral to "they/them". For example "They talk very fast and incomprehensibly to the NPCs before turning to dust." so if the issue is hardcoded based on predefined entries it'd be easier to not mix up pronouns.

I've noticed several times with male selected the generated data will transition from **He** to **She**, or only have female pronouns.

### PR DESCRIPTION
**Pull request by Cellule**
Simplify gender attribution and reduce duplication Removed "medusa" table and inline it in race instead, change stats attribution to medusa to compensate for default female stats modifications.

Closes #2 
I looked as hard as I could and couldn't find other "gender" inconsistencies other than that one

- - -
This change is [Reviewable](https://reviewable.io/reviews/cellule/npc-generator/6)

### CODE REVIEW

**Review by Cellule — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cellule/npc-generator/6)* status: 0 of 7 files reviewed, all discussions resolved

___
*[`src/tables/medusa.json` line 1 at r1](https://reviewable.io/reviews/cellule/npc-generator/6#-NF5eoan4UJE7MRoeAnR:-NF5eoan4UJE7MRoeAnS:by60646) ([raw file](https://github.com/cellule/npc-generator/blob/38e893cad971bc89298a63fe2385a36f19b360b5/src/tables/medusa.json#L1)):*
> ```JSON
> [{ "w": 1, "v": "{$gender=female}{$majPoss=Her }{$minPoss=her }{$majPro=She }{$minPro=she }{$workGender=woman}{$minHim=her}{$beard=}{medusahair}" }]
> ```

urgg I just noticed that `female` also reduces height by 10...
https://github.com/Cellule/npc-generator/blob/c78464a1e14be9a2e27fdf5161e636c85316b105/src/tables/female.json#L4

I might split base female (like gender pronouns and such) from the stats changes

**Review by Cellule — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cellule/npc-generator/6)* status: 0 of 7 files reviewed, all discussions resolved

___
*[`src/tables/medusa.json` line 1 at r1](https://reviewable.io/reviews/cellule/npc-generator/6#-NF5eoan4UJE7MRoeAnR:-NF5f76B4QHBjGVwnmga:bwpqkwu) ([raw file](https://github.com/cellule/npc-generator/blob/38e893cad971bc89298a63fe2385a36f19b360b5/src/tables/medusa.json#L1)):*
<details><summary><i>Previously, Cellule (Michael "Mike" Ferris) wrote…</i></summary><blockquote>

urgg I just noticed that `female` also reduces height by 10...
https://github.com/Cellule/npc-generator/blob/c78464a1e14be9a2e27fdf5161e636c85316b105/src/tables/female.json#L4

I might split base female (like gender pronouns and such) from the stats changes
</blockquote></details>

@etiens what do you think? Are Medusas same height as men or should they be shorter
Basically, should I revert it, so I don't change this stat or is it fine to make them shorter?

**Comment by etiens** [comment]
It's fine if Medusa are shorter, after all they were human or elf females that got cursed to become medusas, so it makes sense that they'd have the same height.
